#!/usr/bin/env python
PACKAGE = "visual_servo_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Kp", double_t, 0, "Kp for y axis", 0.0, -40, 40)
gen.add("Kd", double_t, 0, "Kd for y axis", 0.0, -2, 2)

gen.add("Kf_r0", double_t, 0, "Kalman filter R", 0.0, -1, 1)
gen.add("Kf_q0", double_t, 0, "Kalman filter Q", 0.0, -1, 1)

gen.add("FIR_gain", double_t, 0, "Low pass filter gain", 0.0, -3, 3)

exit(gen.generate(PACKAGE, "visual_servo_control", "tuning"))
